packages/esm-patient-vitals-app/src/vitals/vitals-overview.test.outdated.tsx

```diff
@@ -1,54 +1,107 @@
 import React from "react";
-import VitalsOverview from "./vitals-overview.component";
 import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
 import { BrowserRouter } from "react-router-dom";
 import { of } from "rxjs/internal/observable/of";
-import { mockVitalData } from "../../../../__mocks__/vitals.mock";
+import { mockVitalData } from "../../__mocks__/vitals.mock";
```

Comment:
Well, all mock should be aggregated in the top-level mocks dir. Can you change this?

Reply:
Fixed